tests/unit/e_db_mysqlTest.php

```diff
@@ -286,17 +286,18 @@ public function testUpdate()
 			$db->delete('tmp');
 
 			// Test 1
-			$expected = $db->update('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'test 1', 'WHERE' => 'tmp_ip="127.0.0.1"'));
+			$expected = $db->update('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'Update test 1', 'WHERE' => 'tmp_ip="127.0.0.1"'));
 			$this->assertEmpty($expected, "Test 1 update() failed (not empty {$expected})");
 
+			$actual = 1;
 			// Test 2
-			$actual = $db->insert('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'test 2'));
-			$expected = $db->update('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'test 2a', 'WHERE' => 'tmp_ip="127.0.0.1"'));
-			$this->assertEquals(1, $expected, "Test 2 update() failed ({$actual} != {$expected}");
+			$db->insert('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'test 2'));
+			$expected = $db->update('tmp', array('tmp_ip' => '127.0.0.1', 'tmp_time' => time(), 'tmp_info' => 'Update test 2a', 'WHERE' => 'tmp_ip="127.0.0.1"'));
+			$this->assertEquals($actual, $expected, "Test 2 update() failed ({$actual} != {$expected}");
 
 			// Test 3
-			$expected = $db->update('tmp', 'tmp_ip = "127.0.0.1", tmp_time = tmp_time + 1, tmp_info = "test 3" WHERE tmp_ip="127.0.0.1"');
-			$this->assertEquals(1, $expected, "Test 3 update() failed ({$actual} != {$expected}");
+			$expected = $db->update('tmp', 'tmp_ip = "127.0.0.1", tmp_time = tmp_time + 1, tmp_info = "Update test 3" WHERE tmp_ip="127.0.0.1"');
+			$this->assertEquals($actual, $expected, "Test 3 update() failed ({$actual} != {$expected}");
```

Comment:
In assertions that take both an expected value and an actual value, the expected value is the first argument.

Reply:
Resolved in 77374ae03c568030373f57d57bd7f30cd7d55588